package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.3",
   "homepage": "https://github.com/aframevr/aframe-core",
   "license": "MIT",
-  "main": "src/index.js",
+  "main": "dist/aframe-core.js",
```

Comment:
since the [`README`](https://github.com/aframevr/aframe-core/tree/dev#developing-alongside-projects) still references how to link projects together, how should we change our docs to reflect the new dev workflows?

we didn't really conclude with a decent solution here: https://github.com/aframevr/aframe-core/issues/699#issuecomment-166467822

agreed this is super important; let's figure out how we can get this working. it's arguably more important to get  `aframe-core` to be requirable without issues for other third-party projects - and work on our dev workflow issues shortly thereafter. if + when `aframe` consumes `aframe-core`, this becomes less of an issue. when do we want to do that? and, beyond that, if some project wants to dev against `aframe` what would be the documented steps for folks to get the equivalent of `npm link` (i.e., the `watchify` commands for example)?